rtmp/src/main/java/com/github/faucamp/simplertmp/io/RtmpConnection.java

```diff
@@ -739,4 +761,8 @@ public void setAuthorization(String user, String password) {
   public void setLogs(boolean enable) {
     isEnableLogs = enable;
   }
+
+  public void addMetaData(Object kvp) {
+    addlMetaData = kvp;
```

Comment:
Actually this would overwrite previously added metadata. Works for us, but it probably isn't everyone's use case